### ISSUE TITLE
Check if travis works with newly released setuptools

### ISSUE THREAD

**Issue by grzanka** [DataMedSci/pymchelper#146]
On 3.12.2016 setuptools 30.0.0 were relased. Support for python 3.2 is dropped.

See:
https://github.com/DataMedSci/beprof/pull/104#issuecomment-264649564

**Comment by grzanka** [comment]
There is a problem:
https://travis-ci.org/DataMedSci/pymchelper/jobs/180979209

```
The command tox --notest exited with 1.
Log:
Traceback (most recent call last):
  File "/home/travis/virtualenv/python3.2.6/bin/tox", line 7, in <module>
    from tox import cmdline
  File "/home/travis/virtualenv/python3.2.6/lib/python3.2/site-packages/tox/__init__.py", line 33, in <module>
    from tox.session import main as cmdline  # noqa
  File "/home/travis/virtualenv/python3.2.6/lib/python3.2/site-packages/tox/session.py", line 15, in <module>
    from tox.venv import VirtualEnv
  File "/home/travis/virtualenv/python3.2.6/lib/python3.2/site-packages/tox/venv.py", line 8, in <module>
    from .config import DepConfig, hookimpl
  File "/home/travis/virtualenv/python3.2.6/lib/python3.2/site-packages/tox/config.py", line 9, in <module>
    import pkg_resources
  File "/home/travis/virtualenv/python3.2.6/lib/python3.2/site-packages/pkg_resources/__init__.py", line 78, in <module>
    raise RuntimeError("Python 3.3 or later is required")
RuntimeError: Python 3.3 or later is required
/home/travis/build.sh: line 151:  3949 Terminated              travis_jigger $! $timeout $cmd
```

**Comment by grzanka** [comment]
@ant6 can you also fix this issue applying same fix as in https://github.com/DataMedSci/beprof/pull/104#issuecomment-264649564 ?

**Comment by ant6** [comment]
@grzanka sure

**Comment by grzanka** [comment]
Fixed by PR #147
